packages/types/src/signer.ts

```diff
@@ -16,13 +17,12 @@ export interface Signer {
     txData: string
   ): Promise<SignedTx | undefined>;
   encodeBonding(args: BondingProps): Promise<string | undefined>;
+  submitBond(args: SubmitBondProps): Promise<void>;
```

Comment:
I started to think about this and I am wondering if SubmittingBond is best off here? Like in the future if we keek on adding more stuff like SubmitTransfer, SubmitProposal, and more here, it feels as Signer, do not sound like a most natural place to contain them 🤔

Reply:
It was just the easiest thing to do for now. We will still have to think about how Signer looks, because it's different in different extensions.

Reply:
Yeah, exactly. In Keplr, the signing client also has functionality to submit a transaction. Might not make the most sense here, but I think it's fine for now!